src/env_properties.h

```diff
@@ -96,6 +96,7 @@
   V(changes_string, "changes")                                                 \
   V(channel_string, "channel")                                                 \
   V(chunks_sent_since_last_write_string, "chunksSentSinceLastWrite")           \
+  V(clientId_string, "clientId")                                               \
```

Comment:
For consistency, these should use lower-case, snake-case
```suggestion
  V(client_id_string, "clientId")                                              \
```